server/testdriver/core/src/main/java/org/infinispan/server/test/core/EmbeddedInfinispanServerDriver.java

```diff
@@ -65,6 +65,20 @@ protected void start(String name, File rootDir, File configurationFile) {
       }
    }
 
+   @Override
+   public void startAdditionalServer(int expectedClusterSize) {
+      // expectedClusterSize is not used as there could be other managers not managed by us in the cluster
```

Comment:
When do we expect this to happen?

Reply:
Embedded currently only verifies by assuming all managers it controls are the ones in the view. Due to the manager being from different drivers, we can't verify that. Let me reword this. The forked driver is a similar thing. We would have to change these both to check in different ways. But due to them not requiring this currently, I didn't work on them.

Reply:
Oops, I meant to remove these methods from forked and embedded. If you notice there is no interface for these methods anymore, so this just goes away.